app/services/api_consumption/models/company/api_entreprise.rb

```diff
@@ -73,7 +73,7 @@ def date_de_creation
 
     # Ex : 6202A
     def naf_code
-      activite_principale["code"].delete('.')
+      activite_principale["code"].delete('.') if activite_principale["code"].present?
```

Comment:
C'est plus simple comme ça non ?
```suggestion
      activite_principale["code"]&.delete('.') 
```

Reply:
Mais moins clair ? Je trouve la version longue + explicite sur le fait que la clé `code` peut ne pas être présent